### PR TITLE
feat: add homepage

### PR DESCRIPTION
**Pull request by webdeveloper001**
### Ticket
https://www.notion.so/thegoodparty/

### CODE REVIEW

File: app/components/home/ChallengersSection/ChallengerAvatar.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	border-radius: 20px;
+`;
+const ChallengerAvatar = ({ avatar, party, ...props}) => {
+	let PartyImg;
+	if(party[0] === 'D') {
+		PartyImg = DemocratImg;
+	} else if(party[0] === 'R') {
+		PartyImg = RepublicanImg;
+	}
+  return (
```
Comment (tomer-tgp) [inline app/components/home/ChallengersSection/ChallengerAvatar.js:36]: There should be 5 options here

Comment (tomer-tgp) [reply app/components/home/ChallengersSection/ChallengerAvatar.js:36]: look at the `partyResolver` under electionsHelper

```
if (partyLetter === 'D') {
    return 'DEMOCRAT';
  }
  if (partyLetter === 'R') {
    return 'REPUBLICAN';
  }
  if (partyLetter === 'GP') {
    return 'GREEN PARTY';
  }
  if (partyLetter === 'LP' || partyLetter === 'L') {
    return 'LIBERTARIAN';
  }
  if (partyLetter === 'I') {
    return 'INDEPENDENT';
  }
  if (partyLetter === 'W') {
    return 'AS A WRITE-IN';
  }

```

File: app/components/home/ChallengersSection/ChallengerItem.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        </Grid>
+      </Grid>
+      <NeededVotesWrapper>
+        <NeededVotes>
+          {neededVotes}
+        </NeededVotes>
+        <LineWrapper>
+          <FullLine />
+          <PercentLine percent={neededPercent + '%'} />
+        </LineWrapper>
```
Comment (tomer-tgp) [inline app/components/home/ChallengersSection/ChallengerItem.js:110]: Why not reuse our Supporters bar component?

File: app/components/home/ChallengersSection/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <ChallengersSectionWrapper>
+      <SectionTitle>Good Challengers for 2020</SectionTitle>
+      <SectionDescription>
+        Good candidates of all stripes are challenging the status quo with fresh
+        ideas, not money. We’re mobilizing voters to join these candidates’
+        voting blocs to see if we can get them enough votes to win!
+      </SectionDescription>
+      <ChallengersList>
+        {showedChallengers.map(challenger => (
+          <ChallengerItem challenger={challenger} />
```
Comment (tomer-tgp) [inline app/components/home/ChallengersSection/index.js:52]: you will need a key on the ChallengerItem

File: app/components/home/HomePageWrapper/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import MoneyCorruptionSection from '../MoneyCorruptionSection';
+import TellMeMoreSection from '../TellMeMoreSection';
+import StatsSection from '../StatsSection';
+import RockTheVoteSection from '../RockTheVoteSection';
+import CreatorsSection from '../CreatorsSection';
+import BottomSection from '../BottomSection';
+import ChallengersSection from '../ChallengersSection';
+
+const ContentWrapper = styled.div`
+	&& {
```
Comment (tomer-tgp) [inline app/components/home/HomePageWrapper/index.js:14]:
```
 && {
```
is not necessary for a .div, only for material-ui

File: app/components/home/HomePageWrapper/index.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+			</ContentWrapper>
+			<GrayWrapper>
+				<ContentWrapper className="gray">
+					<RockTheVoteSection />
+				</ContentWrapper>
+			</GrayWrapper>
+			<ContentWrapper>
+				<ChallengersSection challengers={goodChallengers} />
+				<CreatorsSection />
+				{/* <BottomSection /> */}
```
Comment (tomer-tgp) [inline app/components/home/HomePageWrapper/index.js:47]: Not sure why this is commented out, but if you don't want it right now, comment out the import as well please

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

File: app/components/home/ChallengersSection/ChallengerItem.js (hunk 2 above)
Comment (webdeveloper001) [reply app/components/home/ChallengersSection/ChallengerItem.js:110]: it looks like existing Supporters is a little different in the home design.

File: app/components/home/ChallengersSection/ChallengerAvatar.js (hunk 1 above)
Comment (tomer-tgp) [reply app/components/home/ChallengersSection/ChallengerAvatar.js:36]: Lisa Savage is an indepenent

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**